lib/Core/Site/Values/Content.php

```diff
@@ -36,6 +54,11 @@ public function __construct(array $properties = [])
             unset($properties['_fields_data']);
         }
 
+        if (array_key_exists('site', $properties)) {
+            $this->site = $properties['site'];
```

Comment:
This is handled by the parent constructor, so the entire `if` here is superflous, I think.